### DISCUSSION TITLE
Webwhiz Configuration help needed

### DISCUSSION THREAD

**Discussion by b1onix** [webwhiz-ai/webwhiz#145] [Q&A]
Hi there webwhiz community,
i'm kind of having an issue when i try to deploy webwhiz onto my Ubuntu VPS, i deployed it with docker and and when i signin and create a chatbot, and try putting the <script> tag onto my website, the chatbot doesn't show up.. since i deployed onto my subdomain, i put widgeturl (subdomain:port) into the <script> tag but it doesn't work, any assistance would be much appreciated.

**Comment by hrithikt** [comment]
Hi @b1onix ,

Could you check whether you are able to access the bot using the 'Public Chat Interface'?
